src/views/Farms/Farms.tsx

```diff
@@ -142,7 +142,7 @@ const Farms: React.FC = () => {
   const sortFarms = (farms: FarmWithStakedValue[]): FarmWithStakedValue[] => {
     switch (sortOption) {
       case 'apr':
-        return orderBy(farms, 'apy', 'desc')
+        return orderBy(farms, (farm: FarmWithStakedValue) => Number(farm.apy), 'desc')
```

Comment:
I think it would better to ensure that `apy` is the correct type to begin with. @cbunnydev what do you think?

Reply:
farm.apy.toNumber() instead of Number(farm.apy)